examples/ent/schema-viz.html

```diff
@@ -118,20 +118,44 @@
       container.appendChild(tbl);
       return container;
     }
+
     
     const entGraph = JSON.parse("{\"nodes\":[{\"id\":\"Car\",\"fields\":[{\"name\":\"nickname\",\"type\":\"string\"},{\"name\":\"brand\",\"type\":\"string\"},{\"name\":\"model_year\",\"type\":\"int\"}]},{\"id\":\"Pet\",\"fields\":null},{\"id\":\"Post\",\"fields\":null},{\"id\":\"User\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"email\",\"type\":\"string\"},{\"name\":\"role\",\"type\":\"int8\"},{\"name\":\"created\",\"type\":\"time.Time\"},{\"name\":\"age\",\"type\":\"int\"}]}],\"edges\":[{\"from\":\"User\",\"to\":\"Pet\",\"label\":\"pets\"},{\"from\":\"User\",\"to\":\"Post\",\"label\":\"posts\"},{\"from\":\"User\",\"to\":\"User\",\"label\":\"parent\"},{\"from\":\"User\",\"to\":\"Car\",\"label\":\"cars\"}]}");
-    entGraph.nodes = entGraph.nodes.map((n) =>
-      Object.assign(n, {
-        label: n.id,
-        color: nameToColor(n.id),
-        title: fieldsToTable(n.fields),
-      })
-    );
+    const nodes = new vis.DataSet(entGraph.nodes.map((n) =>
```

Comment:
```suggestion
    const nodes = new vis.DataSet(entGraph.nodes.map(n =>
```

Prefer a consistent style (e.g. line 136).